lib/cli/src/commands/package/mod.rs

```diff
@@ -0,0 +1,16 @@
+mod build;
+mod download;
+mod unpack;
+
+pub use {download::CmdPackageDownload, unpack::CmdPackageUnpack};
+
+/// Package related commands.
+#[derive(clap::Subcommand, Debug)]
+// Allowing missing_docs because the comment would override the doc comment on
+// the command struct.
+#[allow(missing_docs)]
+pub enum CmdPackage {
+    Download(CmdPackageDownload),
+    Unpack(unpack::CmdPackageUnpack),
+    Build(build::CmdPackageBuild),
+}
```

Comment:
We don't need the `Cmd` prefix.

```suggestion
pub enum Package {
    Download(PackageDownload),
    Unpack(unpack::PackageUnpack),
    Build(build::PackageBuild),
}
```